src/Operation/ListDatabaseNames.php

```diff
@@ -72,7 +72,6 @@ public function __construct(array $options = [])
      * Execute the operation.
      *
      * @see Executable::execute()
-     * @param Server $server
      * @return Iterator
```

Comment:
`execute()` here already uses an `Iterator` type hint. This will probably come up when searching for useless `@return` annotations that only include a single type. Unions might still be relevant if we use those anywhere.

Reply:
This class is also protected by the BC promise, so return types will have to wait until 2.0.